docs/modules/ROOT/pages/installation/minikube.adoc

```diff
@@ -6,14 +6,8 @@ Follow the instructions in the https://github.com/kubernetes/minikube#installati
 
 Start a new instance of Minikube using the command:
 
-```
-minikube start
-```
-
-After the startup process is completed, you need to **enable the `registry` addon**:
```

Comment:
Putting it in one command is great, but it's better to preserve the instruction that the `registry` addon is necessary. Maybe we can pushing this sentence one-line above?